kore/app/parser/Main.hs

```diff
@@ -164,21 +103,23 @@ main = handleTop $ do
                         $ toVerifiedDefinition indexedModules
                 else putDoc (debug parsedDefinition)
 
-            let KoreParserOptions { patternFileName } = koreParserOptions
-            unless (null patternFileName) $ do
-                parsedPattern <- mainPatternParse patternFileName
-                when willVerify $ do
-                    let KoreParserOptions { mainModuleName } = koreParserOptions
-                    indexedModule <-
-                        lookupMainModule
-                            (ModuleName mainModuleName)
-                            indexedModules
-                        & lift
-                    _ <- mainPatternVerify indexedModule parsedPattern
-                    return ()
-                let KoreParserOptions { willPrintPattern } = koreParserOptions
-                when willPrintPattern $
-                    lift $ putDoc (debug parsedPattern)
```

Comment:
This is just a code refactoring so that it uses the new `KoreParserOptions` type that I introduced